flake.nix

```diff
@@ -133,7 +137,7 @@
             1; # to run the GUI examples with NVIDIA's closed source drivers
 
           shellHook = ''
-            export LLVM_SYS_${llvmMajorMinorStr}_PREFIX="${llvmPkgs.llvm.dev}"
+            export LLVM_SYS_180_PREFIX="${llvmPkgs.dev}"
```

Comment:
The LLVM MajorMinor is actually `181`, but most references still want `180` so we hardcode it here.